metaflow/plugins/aws/batch/batch_decorator.py

```diff
@@ -136,6 +137,9 @@ def step_init(self, flow, graph, step, decos, environment, datastore, logger):
                     if not (my_val is None and v is None):
                         self.attributes[k] = str(max(int(my_val or 0), int(v or 0)))
         self.run_time_limit = get_run_time_limit_for_task(decos)
+        if self.run_time_limit < 60:
```

Comment:
What is the reason for this?

Reply:
AWS Batch has a lower limit of 60 seconds for timeouts. The job submission gets rejected otherwise - which is needed for AWS SFN.